### ISSUE TITLE
Can't play AI anymore when a shell is destroyed

### ISSUE THREAD

**Issue by stonedDiscord** [tgstation/tgstation#38443]
[Round ID]: # terry
[Round ID]: # 89350

[Testmerges]: # e96b3b438cb5d042047baf59eb90bba2bd3050d9

[Reproduction]: # You can't play as AI anymore when one of your cyborg shells get destroyed while you control it.
1. Play as AI
2. Have 2 cyborg AI shells
3. Currently control a med cyborg
4. Try to kill HoP
5. Get hit by a grenade

Problem Description: I ended up being disconnected from the shell, but with the screen locked on disposals and the wireless module offline. You can't interact with anything anymore. No cameras, no reconnecting to another borg nor making vox announcement.

What did you expect to happen: The shell is destroyed and i'm back at my core in AI view.

What happened instead: Screen stuck in AI view on disposals. When I checked on the AI core as a ghost, it was still there and fully functional.

What are the consequences: Can't play AI anymore.

Or is this the same as #38229 and intended?

**Comment by GunHog** [comment]
No, it is not intended. If your shell "dies", then you are supposed to be returned to your AI core.

**Comment by CitrusGender** [comment]
Something has been fucky with cyborgs recently.
